sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/UpdateTrack.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.appcenter.distribute;
+
+/**
+ * Track to use for in-app updates.
+ */
+public enum UpdateTrack {
```

Comment:
Let's use just int with annotations (like in other places) verification to easier wrap it into another languages

Reply:
True, though annotations were not supposed to be public as we depend on a package for that. We can use annotations only for internal code, not for the public signature.

Reply:
We discussed some spec changes anyway and will include the change to int to the PR.

Reply:
Actually we return that value with a getter so it's not a good thing for the customer. It's ok if it makes bindings slightly more difficult, it's more important that the API is easy to use in Java.